microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -6,19 +6,36 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: pwa
-ms.date: 08/04/2023
+ms.date: 08/10/2023
 ---
 # Build PWA-driven widgets
 
 Various operating systems have widgets dashboards that let users read content and perform tasks. Examples of this include Android Home Screen widgets, macOS Dashboard and Today Panel widgets, the Apple Touch Bar, Samsung Daily Cards, Mini App widgets, and smart watch app companions.
 
-On Windows 11, widgets appear in the widgets board which you open from the left corner of your taskbar:
+On Windows 11, widgets appear in the Widgets board which you open from the left corner of your taskbar:
 
-![The widgets board in Windows 11](./widgets-images/windows11-widgets.png) 
+![The Widgets board in Windows 11](./widgets-images/windows11-widgets.png) 
 
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
+<!-- ------------------------------ -->
+#### Requires building a custom widget for the PWA
+
+An existing PWA cannot simply be placed into the widget dashboard as-is, like you can with the Microsoft Edge Sidebar.  Instead, you need to build a custom widget experience that's appropriate for the widget host, which currently is the Win11 Widgets Board.  (There may be other widget hosts in the future.)  The Win11 Widgets Board requires widgets to be built by using Adaptive Card templates instead of HTML and JavaScript, so the widget has to be designed separately from the rest of the app's UI.
+
+For a PWA-driven widget, you build the widget experience by using the web app manifest, Adaptive Card templates, and JavaScript, rather than putting an existing PWA into a widget.  A widget is a separate experience, distinct from PWAs in the sidebar, which takes the entire PWA and puts it in the sidebar.
```

Comment:
I think all the info in the paragraph was covered by the previous one, so do you want to remove it for brevity?

Reply:
removed, was wondering/anticipating, thx